### ISSUE TITLE
Please try a different name?

### ISSUE THREAD

**Issue by iLevye** [junedomingo/react-native-rename#97]
Why it doesn't accept the name?

```
$ react-native-rename "Name" -b com.name.and
Please try a different name.

```

**Comment by cnazha** [comment]
I had the same problem when trying to rename a project with the same name (was trying to rename bundler identifier), I simply renamed it to a different name then back to the original.

First use temporary name:
`npx react-native-rename "NameX" -b com.name.app`

Revert to old name:
`npx react-native-rename "Name" -b com.name.app`

This will probably solve the issue of having to rename an existing project name but change the bundler identifier

**Comment by jvinai** [comment]
Nice workaround but i would love to have a way to just change the bundle ID (we need to do it depending of the kind of build we want to do...)

**Comment by voxspox** [comment]
+1   I just want to modify the bundleId

**Comment by Willt123** [comment]
Any chance we could run it with a --force flag to ignore the sanity check?

**Comment by vomchik** [comment]
Fixed in the last release v2.7.0
